platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/ExtendingConfigurationsIntegrationTest.groovy

```diff
@@ -206,16 +205,8 @@ task checkResolveParentThenChild {
         """)
 
         expect:
-        executer.expectDeprecationWarning("Configuration 'conf2' in project ':project2' extends configuration 'conf1' in project ':project1'. This behavior has been deprecated. This behavior is scheduled to be removed in Gradle 9.0. Configurations can only extend from configurations in the same project. Consult the upgrading guide for further information: https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_8.html#extending_configurations_in_same_project")
-        succeeds ':project2:resolvableConfigurations', '--all'
-        outputContains("""
---------------------------------------------------
-Configuration conf2
---------------------------------------------------
-
-Extended Configurations
-    - conf1
-""")
+        fails ':project2:resolvableConfigurations', '--all'
+        failure.assertHasCause("configuration ':project2:conf2' in project ':project2' cannot extend configuration ':project1:conf1' from project ':project1'. Configurations can only extend from configurations in the same context.")
```

Comment:
🤔 Why say "same context" instead of "same project"?

Reply:
There are configurations in  non-project contexts

Reply:
This test is projects, though.  It would be better to be explicit about what the context is, and say projects when projects, buildscripts when buildscripts, etc.  Right now it might be not clear what this means.